src/neo-vm/ErrorHandle.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+namespace Neo.VM
+{
+    public class FaultState
+    {
+        public bool HoldError;
```

Comment:
Change to `Handled` ? (like https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.keypresseventargs.handled?view=netframework-4.8)

Reply:
HoldError is for record a error and wait endfinally to rethrow it.